stacks/api/bucket/services/bucket.service.ts

```diff
@@ -7,8 +7,8 @@ import {
   FilterQuery,
```

Comment:
Shall we use `BaseCollection` here as well?  Of course, if it is not going to take too much time.

Reply:
Just like bucket-data service, it won't.

Reply:
The hook module uses this service. We need to refactor that module also or we won't prefer BaseCollection for only this service.